cli/src/util/spm.ts

```diff
@@ -35,6 +35,7 @@ export async function generatePackageFile(
 ): Promise<void> {
   const packageSwiftFile = await findPackageSwiftFile(config);
   try {
+    logger.warn('SPM Support is still experimental');
```

Comment:
this is shown on every update/sync, do we want that?
or should only be shown on `npx cap add`?

Reply:
I could see it either way.  I don't think it would hurt even on sync to remind users this is still experimental, especially if they run into issues.

Reply:
@dallastjames What do you think?

Reply:
Yeah - I put it on sync to remind people that they're in the danger zone.